### PR TITLE
Create humans.txt

### PR DESCRIPTION
**Pull request by stevensona**
In support of #124 and Hacktoberfest 2016.

### CODE REVIEW

**Comment by willdurand** [comment]
Hi @stevensona, thanks for your work! I have a few remarks:

* could you please remove the web and email addresses?
* could you please change the `contributors` section with:

```
/* THANKS */
Contributors: https://github.com/TailorDev/monod/graphs/contributors
```

Also, maybe only use `/* TEAM */`  instead of `core team`

**Comment by willdurand** [comment]
Great! Now, could you please squash your commits maybe?

File: humans.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+/* TEAM */
+  William Durand
```
Comment (jmaupetit) [inline humans.txt]: Can you please remove those two spaces at the beginning of each line from this section?

**Comment by willdurand** [comment]
Hey @stevensona, why did you close your PR? It was almost done!

**Pull request closed without merging**